pom.xml

```diff
@@ -401,6 +413,7 @@
               <configuration>
                 <nexusUrl>https://oss.sonatype.org</nexusUrl>
                 <serverId>sonatype.release</serverId>
+                <stagingProfileId>655dc88dc770c3</stagingProfileId>
```

Comment:
@chtyim I think you mentioned that this should not not be there for sonatype - can you clarify? Thanks

Reply:
It is correct. It needs to be removed, otherwise the release build will fail to publish artifacts.